README.md

```diff
@@ -29,17 +29,17 @@ This GitHub Action instruments your workflows using [Honeycomb's buildevents too
 ### Single Job Workflow
 
 ```yaml
-- uses: honeycombio/gha-buildevents@v1
+- uses: honeycombio/gha-buildevents@v2
   with:
     # Required: a Honeycomb API key - needed to send traces.
     apikey: ${{ secrets.BUILDEVENT_APIKEY }}
 
     # Required: the Honeycomb dataset to send traces to.
     dataset: gha-buildevents_integration
 
-    # Optional: status, this will be used in the post section and sent
-    # as status of the trace. Set on the final job of a workflow to signal for the trace to # end
-    # job-status has been deprecated
+    # Required on the final job: status, this will be used in the post section and sent
+    # as status of the trace. Set on the final job of a workflow to signal for the trace # to end
```

Comment:
```suggestion
    # as status of the trace. Set on the final job of a workflow to signal for the trace to end
```

Reply:
We might need actually need the second sentence at all - I think it gets the same message across as the first sentence.